### PR TITLE
#1442 test harness updates

### LINKED ISSUE #1442: Implement test harness sidepanel flyout using Common Properties
ref 1600

Improve test harness by using common properties to manage the config options for the properties side panel

### PR DESCRIPTION
**Pull request by caritaou**
closes #1442 
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: canvas_modules/harness/cypress/integration/canvas/zoom.js
Comment (caritaou) [inline canvas_modules/harness/cypress/integration/canvas/zoom.js:1]: @tomlyn These numbers changed because I modified the size of the side panel

Comment (tomlyn) [reply canvas_modules/harness/cypress/integration/canvas/zoom.js:1]: 👍

**Comment by nmgokhale** [comment]
- After selecting a paramDef from dropdown, properties side panel doesn't close. It closes after clicking "Show properties" button
- `showRequiredIndicator`  - Please change On/Off toggle text to `(required)/(optional)`
- `setDisableRowMoveButtons` - Placeholder should be [{ "name": "parameterName"}] . This config option isn't working.  Tested with example from release - https://github.com/elyra-ai/canvas/releases/tag/release_10.3.0

**Review by nmgokhale — APPROVED** [review]

**Review by matthoward366 — APPROVED** [review]
Looks good

**Pull request merged**